src/components/LicenseRevokeModal/LicenseRevokeModal.scss

```diff
@@ -11,7 +11,7 @@
     }
 
     .license-details {
-      padding: 1rem 1.75rem;
+      padding: 1rem;
```

Comment:
make padding consistent with the alert padding on the left/right per UX feedback.